packages/freesewing.shared/components/workbench/inputs/design-option-count.js

```diff
@@ -53,7 +55,7 @@ const DesignOptionCount = props => {
               min={min}
               max={max}
               setEditCount={setEditCount}
-              t={props.app.t}
+              t={t}
```

Comment:
Going to tackle this later as it's in a couple of places. But I will rewrite it as such.
Will merge it now though, since I don't think this is a blocker, and it would be nice to go into the new week with this merged ;)